src/nodes.coffee

```diff
@@ -3911,6 +3921,13 @@ exports.Existence = class Existence extends Base
       code = "#{code} #{cmp} #{@comparisonTarget}"
     [@makeCode(if o.level <= LEVEL_COND then code else "(#{code})")]
 
+  astType: -> 'UnaryExpression'
```

Comment:
Initially I'd exposed `Existence` as its own unique Coffeescript-specific `type: 'Existence'`

But then realized that conceptually it's a `UnaryExpression` (like eg `!a`), just with a postfix operator. `UnaryExpression` ASTs already have a `prefix` field, so this doesn't even require extending the Babel-style AST at all

Reply:
Sorry I didn't mean literally the same as `!a`, just using `!` as an example where the general semantics is similar to `Existence` (a unary operator whose output is a boolean based on whether its argument is some form of "truthy" or not)

So to be clear, the AST for `!a` looks like
```
type: 'UnaryExpression'
operator: '!'
argument:
  type: 'Identifier'
  name: 'a'
prefix: yes
```
whereas the AST for `a?` looks like
```
type: 'UnaryExpression'
operator: '?'
argument:
  type: 'Identifier'
  name: 'a'
prefix: no
```

Reply:
That makes much more sense. Sounds good 👍